api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
     "# Geneformer for cell class prediction and data projection\n",
     "\n",
-    "This notebook provides examples to utilize the CELLxGENE-maintained fine-tuned Geneformer model with user data. For more information on the model please refer to the [Census model page](https://cellxgene.cziscience.com/census-models).\n",
+    "This notebook provides examples to utilize the CELLxGENE Collaboration fine-tuned Geneformer model with user data. For more information on the model please refer to the [Census model page](https://cellxgene.cziscience.com/census-models).\n",
```

Comment:
What is "CELLxGENE Collaboration"? is that the name of this initiative? Or is Collaboration meant to be lowercased?

Reply:
We had to change the name of this Tier per internal directives.